CHANGELOG.next.asciidoc

```diff
@@ -16,7 +16,6 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 
 *Filebeat*
 
-- Fixed error spam from `add_kubernetes_metadata` processor when running on AKS. {pull}33697[33697]
```

Comment:
I would have expected all of the entries for 8.5.1 and 8.5.2 to have also been removed, are those gone as well

Reply:
You're right. I just corrected it.